packages/x-components/src/components/suggestions/base-suggestion.vue

```diff
@@ -52,6 +51,14 @@
     @Prop({ required: true })
     protected suggestion!: Suggestion;
 
+    /**
+     * The feature from which the events will be emitted.
+     *
+     * @public
+     */
+    @Prop({ required: false }) //TODO: set to true when the suggestions components pass it.
```

Comment:
I really like the TODO, but I prefer to set the Prop as optional and remove the require to false.

```javascript
@Prop() //TODO: add { require: true } when the suggestions components pass it.
protected feature?: QueryFeature;
```

Reply:
I just made it easy for the person to come next (likely me)